evap/templates/base.html

```diff
@@ -5,20 +5,23 @@
 {% load grades_templatetags %}
 {% load results_templatetags %}
 {% load cache %}
+{% load compress %}
 
 <!DOCTYPE html>
 <html>
 <head>
     {% block header %}
-    <title>{% block title %}EvaP{% endblock %}</title>
-    <link rel="stylesheet" href="{% get_static_prefix %}css/jquery.ui.css">
-    <link rel="stylesheet" href="{% get_static_prefix %}css/bootstrap.min.css">
-    <link rel="stylesheet" href="{% get_static_prefix %}css/evap.css">
-    <link rel="stylesheet" href="{% get_static_prefix %}css/select2.min.css" />
-    <link rel="stylesheet" href="{% get_static_prefix %}css/jquery.dataTables.min.css" />
-    <link rel="stylesheet" href="{% get_static_prefix %}css/dataTables.bootstrap.css" />
-    <link rel="stylesheet" href="{% get_static_prefix %}css/jquery.reject.css" />
-    <link rel="shortcut icon" href="{% get_static_prefix %}images/favicon.ico" />
+        <title>{% block title %}EvaP{% endblock %}</title>
+        {% compress css %}
+            <link rel="stylesheet" href="{% get_static_prefix %}css/jquery.ui.css">
+            <link type="text/less" rel="stylesheet" href="{% get_static_prefix %}css/bootstrap.less">
+            <link rel="stylesheet" href="{% get_static_prefix %}css/evap.css">
```

Comment:
i can move it to the bottom, if you test it and tell me everything still looks ok :)